FrameworkSystem/DB/InstalledComponentsDB.py

```diff
@@ -1026,3 +1125,127 @@ def removeInstalledComponents( self, matchFields = {} ):
 
     session.close()
     return S_OK( 'InstalledComponents successfully removed' )
+
+  def addLog( self, newLog ):
+    """
+    Add a new log to the database
+    newComponent argument should be a dictionary with the log fields and
```

Comment:
There's no "newComponent argument". Please also add an example of what can be content of newLog